wolfcrypt/src/asn.c

```diff
@@ -9697,7 +9703,9 @@ static int DecodeBasicOcspResponse(byte* source, word32* ioIndex,
             return ASN_PARSE_E;
 
         InitDecodedCert(&cert, resp->cert, resp->certSz, heap);
-        ret = ParseCertRelative(&cert, CERT_TYPE, VERIFY, cm);
+        /* Don't verify if we don't have access to Cert Manager. */
+        ret = ParseCertRelative(&cert, CERT_TYPE,
```

Comment:
The static function is not called by the OpenSSL compatibility function that requires the verification to be turned off. wolfSSL_d2i_OCSP_RESPONSE() in ocsp.c calls OcspResponseDecode() which calls DecodeBasicOcspResponse() which parses the optional certificate. wolfSSL_d2i_OCSP_RESPONSE() must call OcspResponseDecode().